setup.py

```diff
@@ -26,6 +26,6 @@
     packages=setuptools.find_packages(exclude="tests"),
     package_data={"alarmpy": ["data/*.json"]},
     python_requires=">=3.6",
-    install_requires=["requests", "click"],
+    install_requires=["requests", "click", "paho.mqtt.client"],
```

Comment:
Should probably be an optional dependency

Reply:
Not sure how to do that. Can you please advise?

Reply:
Check out https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies